diff --git a/node_modules/@strapi/admin/admin/src/components/LeftMenu/index.js b/node_modules/@strapi/admin/admin/src/components/LeftMenu/index.js
index 23b936b..30c79a8 100644
--- a/node_modules/@strapi/admin/admin/src/components/LeftMenu/index.js
+++ b/node_modules/@strapi/admin/admin/src/components/LeftMenu/index.js
@@ -84,7 +84,7 @@ const LeftMenu = ({ generalSectionLinks, pluginsSectionLinks }) => {
 
   const menuTitle = formatMessage({
     id: 'app.components.LeftMenu.navbrand.title',
-    defaultMessage: 'Strapi Dashboard',
+    defaultMessage: 'CheapBaltimoreHouses Dashboard',
   });
 
   return (
diff --git a/node_modules/@strapi/admin/admin/src/hooks/useMenu/reducer.js b/node_modules/@strapi/admin/admin/src/hooks/useMenu/reducer.js
index 04f5e6b..1763150 100644
--- a/node_modules/@strapi/admin/admin/src/hooks/useMenu/reducer.js
+++ b/node_modules/@strapi/admin/admin/src/hooks/useMenu/reducer.js
@@ -16,15 +16,6 @@ const initialState = {
       to: '/list-plugins',
       permissions: adminPermissions.marketplace.main,
     },
-    {
-      icon: ShoppingCart,
-      intlLabel: {
-        id: 'global.marketplace',
-        defaultMessage: 'Marketplace',
-      },
-      to: '/marketplace',
-      permissions: adminPermissions.marketplace.main,
-    },
     {
       icon: Cog,
       intlLabel: {
diff --git a/node_modules/@strapi/admin/admin/src/pages/HomePage/HomeHeader.js b/node_modules/@strapi/admin/admin/src/pages/HomePage/HomeHeader.js
index a846220..2fa2112 100644
--- a/node_modules/@strapi/admin/admin/src/pages/HomePage/HomeHeader.js
+++ b/node_modules/@strapi/admin/admin/src/pages/HomePage/HomeHeader.js
@@ -28,33 +28,24 @@ const HomeHeader = ({ hasCreatedContentType, onCreateCT }) => {
             {hasCreatedContentType
               ? formatMessage({
                   id: 'app.components.HomePage.welcome.again',
-                  defaultMessage: 'Welcome 👋',
+                  defaultMessage: 'Welcome to the CheapBaltimoreHouses Admin Panel👋',
                 })
               : formatMessage({
                   id: 'app.components.HomePage.welcome',
-                  defaultMessage: 'Welcome on board!',
+                  defaultMessage: 'Welcome to the CheapBaltimoreHouses Admin Panel👋',
                 })}
           </Typography>
           <WordWrap textColor="neutral600" variant="epsilon">
             {hasCreatedContentType
-              ? formatMessage({
-                  id: 'app.components.HomePage.welcomeBlock.content.again',
-                  defaultMessage:
-                    'We hope you are making progress on your project! Feel free to read the latest news about Strapi. We are giving our best to improve the product based on your feedback.',
-                })
+              ? ''
               : formatMessage({
                   id: 'app.components.HomePage.welcomeBlock.content',
                   defaultMessage:
-                    'Congrats! You are logged as the first administrator. To discover the powerful features provided by Strapi, we recommend you to create your first Content type!',
+                    'Congrats! You are logged as the first administrator. We recommend you to create your first Content type!',
                 })}
           </WordWrap>
           {hasCreatedContentType ? (
-            <Link isExternal href="https://strapi.io/blog">
-              {formatMessage({
-                id: 'app.components.HomePage.button.blog',
-                defaultMessage: 'See more on the blog',
-              })}
-            </Link>
+            ''
           ) : (
             <Button size="L" onClick={onCreateCT} endIcon={<ArrowRight />}>
               {formatMessage({
diff --git a/node_modules/@strapi/admin/admin/src/pages/HomePage/index.js b/node_modules/@strapi/admin/admin/src/pages/HomePage/index.js
index ae1448f..40d1170 100644
--- a/node_modules/@strapi/admin/admin/src/pages/HomePage/index.js
+++ b/node_modules/@strapi/admin/admin/src/pages/HomePage/index.js
@@ -75,14 +75,6 @@ const HomePage = () => {
               />
             </GridItem>
           </Grid>
-          <Grid gap={6}>
-            <GridItem col={8} s={12}>
-              {showGuidedTour ? <GuidedTourHomepage /> : <ContentBlocks />}
-            </GridItem>
-            <GridItem col={4} s={12}>
-              <SocialLinks />
-            </GridItem>
-          </Grid>
         </Box>
       </Main>
     </Layout>
diff --git a/node_modules/@strapi/admin/admin/src/translations/en.json b/node_modules/@strapi/admin/admin/src/translations/en.json
index 52320df..08a64b5 100644
--- a/node_modules/@strapi/admin/admin/src/translations/en.json
+++ b/node_modules/@strapi/admin/admin/src/translations/en.json
@@ -353,7 +353,7 @@
   "app.components.LeftMenu.logout": "Logout",
   "app.components.LeftMenu.logo.alt": "Application logo",
   "app.components.LeftMenu.plugins": "Plugins",
-  "app.components.LeftMenu.navbrand.title": "Strapi Dashboard",
+  "app.components.LeftMenu.navbrand.title": "CheapBaltimoreHouses Dashboard",
   "app.components.LeftMenu.navbrand.workplace": "Workplace",
   "app.components.LeftMenuFooter.help": "Help",
   "app.components.LeftMenuFooter.poweredBy": "Powered by ",
