diff --git a/node_modules/@strapi/admin/admin/src/components/LeftMenu/index.js b/node_modules/@strapi/admin/admin/src/components/LeftMenu/index.js
index 23b936b..30c79a8 100644
--- a/node_modules/@strapi/admin/admin/src/components/LeftMenu/index.js
+++ b/node_modules/@strapi/admin/admin/src/components/LeftMenu/index.js
@@ -84,7 +84,7 @@ const LeftMenu = ({ generalSectionLinks, pluginsSectionLinks }) => {
 
   const menuTitle = formatMessage({
     id: 'app.components.LeftMenu.navbrand.title',
-    defaultMessage: 'Strapi Dashboard',
+    defaultMessage: 'CheapBaltimoreHouses Dashboard',
   });
 
   return (
diff --git a/node_modules/@strapi/admin/admin/src/hooks/useMenu/reducer.js b/node_modules/@strapi/admin/admin/src/hooks/useMenu/reducer.js
index 04f5e6b..1763150 100644
--- a/node_modules/@strapi/admin/admin/src/hooks/useMenu/reducer.js
+++ b/node_modules/@strapi/admin/admin/src/hooks/useMenu/reducer.js
@@ -16,15 +16,6 @@ const initialState = {
       to: '/list-plugins',
       permissions: adminPermissions.marketplace.main,
     },
-    {
-      icon: ShoppingCart,
-      intlLabel: {
-        id: 'global.marketplace',
-        defaultMessage: 'Marketplace',
-      },
-      to: '/marketplace',
-      permissions: adminPermissions.marketplace.main,
-    },
     {
       icon: Cog,
       intlLabel: {
diff --git a/node_modules/@strapi/admin/admin/src/pages/HomePage/HomeHeader.js b/node_modules/@strapi/admin/admin/src/pages/HomePage/HomeHeader.js
index a846220..2fa2112 100644
--- a/node_modules/@strapi/admin/admin/src/pages/HomePage/HomeHeader.js
+++ b/node_modules/@strapi/admin/admin/src/pages/HomePage/HomeHeader.js
@@ -28,33 +28,24 @@ const HomeHeader = ({ hasCreatedContentType, onCreateCT }) => {
             {hasCreatedContentType
               ? formatMessage({
                   id: 'app.components.HomePage.welcome.again',
-                  defaultMessage: 'Welcome 👋',
+                  defaultMessage: 'Welcome to the CheapBaltimoreHouses Admin Panel👋',
                 })
               : formatMessage({
                   id: 'app.components.HomePage.welcome',
-                  defaultMessage: 'Welcome on board!',
+                  defaultMessage: 'Welcome to the CheapBaltimoreHouses Admin Panel👋',
                 })}
           </Typography>
           <WordWrap textColor="neutral600" variant="epsilon">
             {hasCreatedContentType
-              ? formatMessage({
-                  id: 'app.components.HomePage.welcomeBlock.content.again',
-                  defaultMessage:
-                    'We hope you are making progress on your project! Feel free to read the latest news about Strapi. We are giving our best to improve the product based on your feedback.',
-                })
+              ? ''
               : formatMessage({
                   id: 'app.components.HomePage.welcomeBlock.content',
                   defaultMessage:
-                    'Congrats! You are logged as the first administrator. To discover the powerful features provided by Strapi, we recommend you to create your first Content type!',
+                    'Congrats! You are logged as the first administrator. We recommend you to create your first Content type!',
                 })}
           </WordWrap>
           {hasCreatedContentType ? (
-            <Link isExternal href="https://strapi.io/blog">
-              {formatMessage({
-                id: 'app.components.HomePage.button.blog',
-                defaultMessage: 'See more on the blog',
-              })}
-            </Link>
+            ''
           ) : (
             <Button size="L" onClick={onCreateCT} endIcon={<ArrowRight />}>
               {formatMessage({
diff --git a/node_modules/@strapi/admin/admin/src/pages/HomePage/index.js b/node_modules/@strapi/admin/admin/src/pages/HomePage/index.js
index ae1448f..40d1170 100644
--- a/node_modules/@strapi/admin/admin/src/pages/HomePage/index.js
+++ b/node_modules/@strapi/admin/admin/src/pages/HomePage/index.js
@@ -75,14 +75,6 @@ const HomePage = () => {
               />
             </GridItem>
           </Grid>
-          <Grid gap={6}>
-            <GridItem col={8} s={12}>
-              {showGuidedTour ? <GuidedTourHomepage /> : <ContentBlocks />}
-            </GridItem>
-            <GridItem col={4} s={12}>
-              <SocialLinks />
-            </GridItem>
-          </Grid>
         </Box>
       </Main>
     </Layout>
diff --git a/node_modules/@strapi/admin/admin/src/pages/SettingsPage/pages/ApplicationInfosPage/index.js b/node_modules/@strapi/admin/admin/src/pages/SettingsPage/pages/ApplicationInfosPage/index.js
index 1adc22f..3d5429d 100644
--- a/node_modules/@strapi/admin/admin/src/pages/SettingsPage/pages/ApplicationInfosPage/index.js
+++ b/node_modules/@strapi/admin/admin/src/pages/SettingsPage/pages/ApplicationInfosPage/index.js
@@ -116,11 +116,7 @@ const ApplicationInfosPage = () => {
                     </Typography>
                     <Typography as="p">v{strapiVersion}</Typography>
                     <Link
-                      href={
-                        appInfos.communityEdition
-                          ? 'https://discord.strapi.io'
-                          : 'https://support.strapi.io/support/home'
-                      }
+                      href={`mailto:rhuie@altmails.com`}
                       isExternal
                       endIcon={<ExternalLink />}
                     >
@@ -141,42 +137,13 @@ const ApplicationInfosPage = () => {
                       {formatMessage({
                         id: currentPlan,
                         defaultMessage: `${
-                          appInfos.communityEdition ? 'Community Edition' : 'Enterprise Edition'
+                          appInfos.communityEdition ? 'One time payment. No contracted support. Review Terms sent.' : 'One time payment. No contracted support. Review Terms sent.'
                         }`,
                       })}
                     </Typography>
                   </GridItem>
                 </Grid>
 
-                <Grid paddingTop={1}>
-                  <GridItem col={6} s={12}>
-                    {shouldUpdateStrapi && (
-                      <Link
-                        href={`https://github.com/strapi/strapi/releases/tag/${latestStrapiReleaseTag}`}
-                        isExternal
-                        endIcon={<ExternalLink />}
-                      >
-                        {formatMessage({
-                          id: 'Settings.application.link-upgrade',
-                          defaultMessage: 'Upgrade your admin panel',
-                        })}
-                      </Link>
-                    )}
-                  </GridItem>
-                  <GridItem col={6} s={12}>
-                    <Link
-                      href="https://strapi.io/pricing-self-hosted"
-                      isExternal
-                      endIcon={<ExternalLink />}
-                    >
-                      {formatMessage({
-                        id: 'Settings.application.link-pricing',
-                        defaultMessage: 'See all pricing plans',
-                      })}
-                    </Link>
-                  </GridItem>
-                </Grid>
-
                 <Box paddingTop={1}>
                   <Typography variant="sigma" textColor="neutral600">
                     {formatMessage({
diff --git a/node_modules/@strapi/admin/admin/src/translations/en.json b/node_modules/@strapi/admin/admin/src/translations/en.json
index 52320df..ada6141 100644
--- a/node_modules/@strapi/admin/admin/src/translations/en.json
+++ b/node_modules/@strapi/admin/admin/src/translations/en.json
@@ -115,8 +115,8 @@
   "Settings.application.link-pricing": "See all pricing plans",
   "Settings.application.link-upgrade": "Upgrade your admin panel",
   "Settings.application.node-version": "node version",
-  "Settings.application.strapi-version": "strapi version",
-  "Settings.application.strapiVersion": "strapi version",
+  "Settings.application.strapi-version": "CheapBaltimoreHouses Dashboard version",
+  "Settings.application.strapiVersion": "CheapBaltimoreHouses Dashboard version",
   "Settings.application.title": "Overview",
   "Settings.application.customization": "Customization",
   "Settings.application.customization.carousel.title": "Logo",
@@ -353,7 +353,7 @@
   "app.components.LeftMenu.logout": "Logout",
   "app.components.LeftMenu.logo.alt": "Application logo",
   "app.components.LeftMenu.plugins": "Plugins",
-  "app.components.LeftMenu.navbrand.title": "Strapi Dashboard",
+  "app.components.LeftMenu.navbrand.title": "CheapBaltimoreHouses Dashboard",
   "app.components.LeftMenu.navbrand.workplace": "Workplace",
   "app.components.LeftMenuFooter.help": "Help",
   "app.components.LeftMenuFooter.poweredBy": "Powered by ",
@@ -391,8 +391,8 @@
   "app.components.ToggleCheckbox.on-label": "True",
   "app.components.UpgradePlanModal.button": "Learn more",
   "app.components.UpgradePlanModal.limit-reached": "You have reached the limit",
-  "app.components.UpgradePlanModal.text-ce": "Community Edition",
-  "app.components.UpgradePlanModal.text-ee": "Enterprise Edition",
+  "app.components.UpgradePlanModal.text-ce": "One time payment. No contracted support. Review Terms sent.",
+  "app.components.UpgradePlanModal.text-ee": "One time payment. No contracted support. Review Terms sent.",
   "app.components.UpgradePlanModal.text-power": "Unlock the full power of Strapi by upgrading your plan to the Enterprise Edition",
   "app.components.UpgradePlanModal.text-strapi": "of Strapi by upgrading your plan to the",
   "app.components.Users.MagicLink.connect": "Copy and share this link to give access to this user",
